konduit-serving-cli/pom.xml

```diff
@@ -38,7 +38,25 @@
             </dependency>
         </dependencies>
     </dependencyManagement>
+
     <dependencies>
+        <dependency>
```

Comment:
Is this needed? All the javacpp deps should already be a transitive dependency.

Reply:
They are but those deps are in provided scope so they won't be included in the cli jar or you can say that it can't be guaranteed that it's gonna be present